examples/vpc-saperate-private-db-cache-route/main.tf

```diff
@@ -0,0 +1,29 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "vpc" {
+  source = "./terraform-aws-vpc"
+
+  name = "sandbox"
```

Comment:
Call it `vpc-separate-private-route-tables`, also make sure you name directory the same.

Reply:
Done that